llvm/tools/llvm-objdump/llvm-objdump.cpp

```diff
@@ -3533,6 +3533,35 @@ commaSeparatedValues(const llvm::opt::InputArgList &InputArgs, int ID) {
   return Values;
 }
 
+static void mcpuHelp() {
+  std::string Error;
+  Triple DummyTriple;
+
+  if (!TripleName.empty())
+    DummyTriple.setTriple(TripleName);
+  else {
```

Comment:
LLVM style guide says that if braces are required for one part of a conditional sequence, they should be used for all parts.
```suggestion
  if (!TripleName.empty()) {
    DummyTriple.setTriple(TripleName);
  } else {
```